src/OSPSuite.Core/Chart/Axis.cs

```diff
@@ -134,6 +134,13 @@ public IDimension Dimension
          }
       }
 
+      public void Reset()
```

Comment:
This is the actual reset code called when the curve is being removed and axis reseted